tests/md5/md5.cc

```diff
@@ -0,0 +1,374 @@
+/* MD5
+ converted to C++ class by Frank Thilo (thilo@unix-ag.org)
+ for bzflag (http://www.bzflag.org)
```

Comment:
if we have a self-written implementation, i would put it into `Utilities::`. I would actually also move `rand` there. `rand` can be used to add function to deal.II vector to initialize with random numbers, maybe hashes can also be used somewhere later.